.github/git_rules/pr_code_reviewer.py

```
@@ -30,143 +42,156 @@ def load_template(self, template_name):
-
-**Happy Coding! 🚀**"""
+        # 템플릿 사용 (개별 AI 리뷰)
+        review_data = reviews[0]
+        comment_body = template.format(
+            ai_name=review_data['ai_name'],
+            review=review_data['review']
+        )
         
         try:
             self.pr.create_issue_comment(comment_body)
             print(f"✅ 리뷰 코멘트 작성 완료")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)